### DISCUSSION TITLE
ssh키 권한변경시 문의

### DISCUSSION THREAD

**Discussion by ChoiKwangHoon** [tedilabs/fastcampus-devops#12] [Q&A] [answered]
강의 내용중 ssh개인키 권한 변경시 다운로드 받은 디렉토리에서  파워쉘을 이용해 chmod 명령어가 실행이 되지 않습니다.
만약 파워쉘 말고 ubutu쉘을 사용 해야한다면  다운로드 받은 키에 접근을 어떻게 할수 있는지요??

![image](https://user-images.githubusercontent.com/40383973/132478251-60e73de7-ec8f-4f54-99dc-841e7ce5d3db.png)

**Comment by ChoiKwangHoon** [comment]
우분투로 접근후 chmod 권한변경을 해봐도 .. 첨부이미지와 같이 권한 변경이 안됩니다...ㅠ.ㅠ
![image](https://user-images.githubusercontent.com/40383973/132481213-9cc5bb4d-62cd-4089-b877-7f6823ebe7a5.png)

**Comment by ChoiKwangHoon** [comment, marked as answer]
구글링을 통해  저와같은 분이 계셔서 해봤지만 역시 안되네요...
![image](https://user-images.githubusercontent.com/40383973/132481520-83e46c84-8840-4326-88d3-7c14ac5faa6d.png)

**Reply by posquit0** [reply]
윈도우 파일시스템 관련은 건드리지 않는게 좋을 듯 합니다.
[해당 블로그 글](https://m.blog.naver.com/PostView.naver?isHttpsRedirect=true&blogId=nsh83&logNo=221591217177)의 방법처럼 윈도우 내 PEM 키를 WSL 리눅스 내 디렉토리로 복사 후 권한 설정하여 사용해보시기 바랍니다.
